test/ndb_api_helpers_test.go

```diff
@@ -131,17 +134,19 @@ func TestGetOOBProfiles(t *testing.T) {
 	}
 }
 
-func TestGetOOBProfilesOnlyGetsTheSmallOOBComputeProfile(t *testing.T) {
+func TestGetProfilesOnlyGetsTheSmallOOBComputeProfile(t *testing.T) {
```

Comment:
Can we change the name to "TestGetProfilesOnlyGetsTheSmallOOBComputeProfileWhenNoProfileInfoIsProvided" now?